src/Aspire.Hosting/Dashboard/DashboardViewModelService.cs

```diff
@@ -467,7 +467,22 @@ private async Task WriteExecutableChange(ExecutableViewModel executableViewModel
                     && project.GetEffectiveLaunchProfile() is LaunchProfile launchProfile
                     && launchProfile.LaunchUrl is string launchUrl)
                 {
-                    endpointString += $"/{launchUrl}";
+                    if (!launchUrl.Contains("://"))
+                    {
+                        // This is relative URL
+                        endpointString += $"/{launchUrl}";
```

Comment:
Can the launchUrl already be specified with a `/` at the first position? And if so, do we need to treat that was being relative to the root rather than to the end of the endpoint? (e.g. `https://localhost:12345/something` + `/another` should be `https://localhost:12345/another`, not `https://localhost:12345/something/another`). Or is that not a concern with what these values could be?

Reply:
No launchUrl cannot start with `/`. The applicationUrl may be missing trailing `/` but since here we are using endpoint so this should be fine.

Reply:
I also verified that having applicationUrl to be something other than base address (like `https://localhost:12345/something`) throws. The error message asks to use launchUrl or configure in code rather than specify in applicationUrl.